### PR TITLE
added score board

### PR DESCRIPTION
**Pull request by willharrison**
simple score board.  will need to work on it some more but wanted to get it up there.

### CODE REVIEW

File: js/app/state/create.js [hunk 1]
```diff
@@ -15,6 +15,13 @@ define(["app/grid", "app/generator"], function(grid, generator){
     var create = function(game){
         grid.display(game);
         generator.start(game);
-    };
+     
+        var text = "Score: 0";
+        var style = { font: "20px Arial", fill: "#fff", align: "center" };
+
+        this.game.score = 0;
+        this.game.scoreBoard = this.game.add.text(this.game.world.centerX + 150, 27, text, style);
```
Comment (ALSchwalm) [inline js/app/state/create.js]: Rendering text is somewhat expensive, so I think I'd like to do this in html.

Comment (willharrison) [inline js/app/state/create.js]: we can change it later, I just wanted to get something up on it for now.

**Comment by ALSchwalm** [comment]
I think it would be best to have the score system done in html (along with the game time, largest combo and other game info). This system should probably be abstracted into some kind of scoring module

**Comment by ALSchwalm** [comment]
I'm going to go ahead and close this. Feel free to make another PR when you have the scoreboard more fully finished.

**Pull request closed without merging**